app/api/items/get_task_token.go

```diff
@@ -13,38 +12,34 @@ import (
 	"github.com/France-ioi/AlgoreaBackend/app/token"
 )
 
-// swagger:operation GET /items/{item_id}/task-token items itemTaskTokenGet
+// swagger:operation GET /attempts/{attempt_id}/task-token items itemTaskTokenGet
 // ---
 // summary: Get a task token
 // description: >
-//   Get a task token with a refreshed active attempt.
+//   Get a task token with the refreshed active attempt.
 //
 //
 //   * If there is no row for the current user and the given item in `users_items`, the service creates one.
 //
 //   * If the active attempt (`active_attempt_id`) is not set in the `users_items` for the item and the user,
-//   the service chooses the most recent one among all the user's attempts (or the team's attempts if
-//   `items.has_attempts`=1)  for the given item. If no attempts found, the new one gets created and chosen as active.
+//   the service makes `attempt_id` active.
```

Comment:
At the end, we discussed with Michel that it is better that the backend does not auto-switch the active attempt, it allows the user to have 2 windows open on two different attempts without having the active attempt switch all the time.
(update on L25, L19, L29, code and tests)

Reply:
fixed